docs/REDASH.md

```diff
@@ -0,0 +1,8 @@
+Redash access is [in the following link](https://redash.hasadna.org.il/)
+
+In order to grant access to a table for Redash user:
+
+1. Login to the Rancher instance (Ori Hoch / Atalya Alon can grant access)
```

Comment:
What is Rancher ?

Reply:
[Rancher](https://rancher.com/) is the hosting service of our k8

Reply:
@OriHoch should we add the link to our rancher instance to github?

Reply:
no, shouldn't include link to it